docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -282,7 +543,78 @@ data:
     console.log(targetDate);
 ```
 
-### More examples
+## Examples for a custom-runtime container
+
+For an example of a `KeptnTaskDefinition` that defines a custom container.
+ see
+[container-task.yaml](<https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/base/container-task.yaml>.
+The `spec` includes:
+
+```yaml
+spec:
+  container:
+    name: testy-test
+    image: busybox:1.36.0
+    command:
+      - 'sh'
+      - '-c'
+      - 'sleep 30'
+```
+
+This task is then referenced in
+
+[app.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/version-3/app.yaml).
+
+This is a a trivial example that just runs `busybox`,
+then spawns a shell and runs the `sleep 30` command.
+
+## Examples for a python-runtime runner
+
+### Example 1: inline code for a python-runtime runner
+
+You can embed python code directly in the task definition.
+This example prints data stored in the parameters map:
+{{< embed path="/operator/config/samples/python_execution/taskdefinition_pyfunction_inline.yaml" >}}
+
+### Example 2: httpRef for a python-runtime runner
+
+You can refer to code stored online.
+For example, we have a few examples available in the
+[python-runtime samples](https://github.com/keptn/lifecycle-toolkit/tree/main/python-runtime/samples)
+tree.
+
+Consider the following:
+{{< embed path="/operator/config/samples/python_execution/taskdefinition_pyfunction_configmap.yaml" >}}
+
+### Example 3: functionRef for a python-runtime runner
+
+You can refer to an existing `KeptnTaskDefinition`.
+This example calls the inline example
+but overrides the data printed with what is specified in the task:
+{{< embed path="/operator/config/samples/python_execution/taskdefinition_pyfunction_recursive.yaml" >}}
+
+### Example 4: ConfigMapRef for a python-runtime runner
+
+{{< embed path="/operator/config/samples/python_execution/taskdefinition_pyfunction_configmap.yaml" >}}
+
+### Allowed libraries for the python-runtime runner
+
+The following example shows how to use some of the allowed packages, namely:
+requests, json, git, and yaml:
+
+{{< embed path="/operator/config/samples/python_execution/taskdefinition_pyfunction_configmap.yaml">}}
```

Comment:
this should use /operator/config/samples/python_execution/taskdefinition_pyfunction_inline_printargs_py.yaml